docs/dscom.build/ReadMe.md

```diff
@@ -0,0 +1,229 @@
+# dscom.build

```

Comment:
File will be moved to src/dscom.build/doc

Reply:
Done in e36ea6731206b710e736dd335e93233de553801f